crates/subspace-networking/Cargo.toml

```diff
@@ -62,14 +62,25 @@ features = [
     "metrics",
     "noise",
     "ping",
-    "quic",
     "request-response",
     "serde",
     "tcp",
     "tokio",
     "websocket",
     "yamux",
 ]
+[dependencies.libp2p-quic]
+version = "0.8.0-alpha"
+features = ["tokio"]
+
+# Remove after this patch goes to the release branch
+[dependencies.libp2p-kad]
+version = "0.44.3"
+
+# Remove after this patch goes to the release branch
+[dependencies.libp2p-connection-limits]
+version = "0.2.1"
```

Comment:
I'm not sure what these two are doing here. If you needed to update them beyond `0.x.0` then just `cargo update -p package_name` is sufficient or you can do what you did here and then remove them from `Cargo.lock` since they are already updated in lock file.

I also don't know how to interpret those comments. They are not TODOs and did you mean that we wait for that patch version of the crate being dependent on by `libp2p`? If so I explained above that this is not necessary.

Reply:
We discussed that offline: I'd like to make those dependencies go to Cargo.lock even if we for some reason regenerate it.